src/devices/feibit.ts

```diff
@@ -143,6 +143,123 @@ const definitions: Definition[] = [
         fromZigbee: [fz.command_recall],
         exposes: [e.action(['recall_*']), e.switch()],
     },
+    {
+        zigbeeModel: ['FB56+SKT06HM1.1'],
+        model: 'TSKT106W-M1',
+        vendor: 'Feibit',
+        description: 'Smart Socket -T',
+        extend: [deviceEndpoints({endpoints: {'16': 16, '17': 17}}), onOff({powerOnBehavior: false, endpointNames: ['16', '17']})],
```

Comment:
Can we use more friendly names here? e.g. `endpoints: {'left': 16, 'right': 17}`

Reply:
OK~
```js
zigbeeModel: ['FB56+SKT06HM1.1'],
model: 'TSKT106W-M1',
vendor: 'Feibit',
description: 'Smart Socket -T',
extend: [deviceEndpoints({endpoints: {'holes': 16, 'usb': 17}}), onOff({powerOnBehavior: false, endpointNames: ['holes', 'usb']})]
```